app/src/main/res/layout/numeric_input_interaction_item.xml

```diff
@@ -40,6 +43,19 @@
       android:textColor="@color/oppiaPrimaryText"
       android:textColorHint="@color/editTextHint"
       android:textSize="14sp"
-      android:textStyle="italic" />
-  </FrameLayout>
+      android:textStyle="italic"
+      app:textChangedListener="@{viewModel.answerTextWatcher}" />
+
+    <TextView
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="16dp"
+      android:layout_marginEnd="8dp"
+      android:fontFamily="sans-serif"
+      android:minHeight="32dp"
+      android:text="@{viewModel.errorMessage.get()}"
+      android:textColor="@color/editTextError"
+      android:textSize="12sp"
+      android:visibility="@{viewModel.errorMessage.get().length()>0? View.VISIBLE : View.INVISIBLE}" />
```

Comment:
Nit: ``viewModel.errorMessage.get().length() > 0 ? View.VISIBLE : View.INVISIBLE`` (spacing between the condition operators). Ditto also above in the test version of this layout.

Reply:
Done